### PR TITLE
fix:Standardization of folders to kebab-case. modifed home - add avat…

### PR DESCRIPTION
**Pull request by NicolasLLans**
# Description:

Add name and avatar selection. And a function that captures and the data in a json that prints through the console.
Standardization of folders to kebab-case.
Changed the name of the UI folder to components to avoid conflicts when naming the folder that will contain components.

# Trello link:

[https://trello.com/c/7pliEbQS/<Edit This>](https://trello.com/c/FeMnxVdc/65-vista-p%C3%A1gina-de-inicio-y-sala-de-espera)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### CODE REVIEW

**Review by WanderleeDev — APPROVED** [review]

File: src/app/components/avatar-Image/avatar-image.component.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.avatar{
+  width: 150px;
+}
+
+.btn-avatar-change{
+  background-color: var(--game-clr-green) !important;
```
Comment (Alejandro25AR) [inline src/app/components/avatar-Image/avatar-image.component.css:6]: Change variable with --accent-color

File: src/app/components/avatar-Image/avatar-image.component.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+}
+
+.btn-avatar-change:hover{
+  background-color: var(--game-clr-yellow) !important;
```
Comment (Alejandro25AR) [inline src/app/components/avatar-Image/avatar-image.component.css:13]: Normally the use of "!important" is bad practice

File: src/app/components/custom-Button/custom-button.component.css
Comment (Alejandro25AR) [inline src/app/components/custom-Button/custom-button.component.css:1]: Lowercase "Button".

File: src/app/components/avatar-Image/avatar-image.component.css
Comment (Alejandro25AR) [inline src/app/components/avatar-Image/avatar-image.component.css:1]: Lowercase the folder name to "avatar-image".

File: src/app/pages/home/home.component.html [hunk 3]
```diff
@@ -6,22 +6,23 @@
       <div class="card-head p-4">
         <h1 class="text-center title-y">Elegí un avatar y escribí tu nombre</h1>
```
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:7]: Change the title to a more neutral tone to "Selecciona un avatar"".

File: src/app/pages/home/home.component.html [hunk 4]
```diff
@@ -6,22 +6,23 @@
...
-      <div class="card-body p-4">
+      <div class="card-body p-4 justify-content-center">
+        <app-avatar-image [images]="avatarImages" (avatarSelected)="selectAvatar($event)"></app-avatar-image>
         <form class="card-footer flex-fill">
-            <app-avatar-image [images]="avatarImages"></app-avatar-image>
           <input
             type="text"
             class="bg-purpel"
-            placeholder="Escribí tu nombre" />
+            placeholder="Escribí tu nombre"
```
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:15]: Change the placeholder to "Nombre" only.

File: src/app/pages/home/home.component.html [hunk 5]
```diff
@@ -6,22 +6,23 @@
...
-            placeholder="Escribí tu nombre" />
+            placeholder="Escribí tu nombre" 
+            (input)="onNameInputChange($event)"/>
         </form>
       </div>
     </div>
-    <div class="d-flex flex-row justify-content-center gap-4 p-4">
-      <app-custom-button>Iniciar partida</app-custom-button>
+    <div class="d-flex flex-row justify-content-center gap-4">
+      <app-custom-button (click)="iniciarPartida()">Iniciar partida</app-custom-button>
```
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:21]: El texto del botón no describe la acción que realmente realiza, cámbialo por "Crear sala".

File: src/app/pages/home/home.component.ts
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.ts:1]: Possibly there is a conflict with the avatar-image folder as it is named as avatar-Image with the letter "i" capitalized.

File: src/app/components/custom-Button/custom-button.component.css
Comment (Alejandro25AR) [inline src/app/components/custom-Button/custom-button.component.css:1]: Lowercase the folder name to "custon-button".

File: src/app/components/avatar-Image/avatar-image.component.css
Comment (Alejandro25AR) [inline src/app/components/avatar-Image/avatar-image.component.css:1]: Lowercase the folder name to "avatar-image".

File: src/app/pages/home/home.component.html (hunk 3 above)
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:7]: Change the title to a more neutral tone to "Selecciona un avatar"".

File: src/app/pages/home/home.component.html (hunk 4 above)
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:15]: Change the placeholder to "Nombre" only.

File: src/app/pages/home/home.component.html (hunk 5 above)
Comment (Alejandro25AR) [inline src/app/pages/home/home.component.html:21]: The button text does not describe the action it actually performs, change it to "Crear sala".

**Review by Alejandro25AR — CHANGES_REQUESTED** [review]
I have already made the revision and you must correct some things

**Pull request closed without merging**